lib/screens/explore_screens/vouch_for_a_member/vouch_for_a_member_screen.dart

```diff
@@ -39,6 +42,7 @@ class VouchForAMemberScreen extends StatelessWidget {
               context: context,
               barrierDismissible: false,
               builder: (_) {
+                FocusScope.of(context).unfocus();
```

Comment:
This means you end up showing success on top of the search ?

Reply:
changed 👍